test/helpers/accounts.ts

```diff
@@ -0,0 +1,25 @@
+import { DevModeContext } from "@moonwall/cli";
+import { KeyringPair } from "@moonwall/util";
+import { generateKeyringPair, alith } from "../../tests/util/accounts";
+import { MIN_GLMR_STAKING, GLMR } from "../../tests/util/constants";
+
+export async function createAccounts(
```

Comment:
This was also added in `weights.ts` helper (https://github.com/moonbeam-foundation/moonbeam/blob/master/test/helpers/weights.ts#L7). Maybe it's better the remove the one in `weights.ts` and use this one